pkg/cgroup/cgroup_stats_linux.go

```diff
@@ -43,11 +43,14 @@ type CCgroupV12StatManager struct {
 // See comments here: https://github.com/sustainable-computing-io/kepler/pull/609#discussion_r1155043868
 func NewCGroupStatManager(pid int) (CCgroupStatHandler, error) {
 	p := fmt.Sprintf(procPath, pid)
-	_, path, err := cgroups.ParseCgroupFileUnified(p)
+	cgroupMap, path, err := cgroups.ParseCgroupFileUnified(p)
 	if err != nil {
 		return nil, err
 	}
-
+	if path == "" {
+		// if there is no None subsystem (<controller>::<cgrouppath>), use common path from pids subsystem
```

Comment:
There is a typo:
`if there is no None subsystem`
->
`if there is no subsystem`